ebill_paynet/__manifest__.py

```diff
@@ -17,6 +17,7 @@
         "l10n_ch_qriban",
         "queue_job",
         "sale",
+        "delivery",
```

Comment:
wouldn't be better to add a glue module?

Reply:
Yes I think it would be...

Reply:
I have added a comment in the roadmap about it.